src/Compile/CompilePanel.ts

```diff
@@ -154,12 +166,114 @@ export class CompilePanel {
       <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script type="module" src="${toolkitUri}"></script>
         <script type="module" src="${jsUri}"></script>
+        <link rel="stylesheet" href="${cssUri}">
         <title>Compile</title>
       </head>
       <body>
         <h1>Compile</h1>
-        WIP
+        <div class='left-margin-24'>
+          <div class='compile-selections-container'>
+            <div class='text-row'>
+              <span class='two-column-text-left'>
+                    <span>&#8226;</span> Target Code:
+              </span>
+              <span class='two-column-text-right'>
+                <vscode-dropdown id='target-code'>
```

Comment:
For `vscode-dropdown`, refer to https://github.com/Samsung/ONE-vscode/pull/431/files#r839132734